docs/topics/RPC.md

```diff
@@ -171,9 +171,9 @@ Events are payloads sent over the socket to a client that correspond to events i
 
 Used to authenticate a new client with your app. By default this pops up a modal in-app that asks the user to authorize access to your app.
 
-**We currently do not allow access to RPC for unapproved games without an entry on a game's allowlist.** We grant 50 allowlist spots, which should be ample for development and testing. After approval, this restriction is removed and the allowlist is no longer needed.
+**We currently do not allow access to RPC for unapproved games without an entry on a game's testers list.** We grant 50 testing spots, which should be ample for development and testing. After approval, this restriction is removed and the testers feature is no longer needed.
```

Comment:
``and the testers feature is no longer needed.`` might can be lift out i think, or maybe ``and RPC features of your app will be available to all users``